### PR TITLE
YALB-401: Date styling

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
### Relates to the following PRs:
- https://github.com/yalesites-org/yalesites_profile/pull/16
- https://github.com/yalesites-org/component-library-twig/pull/72

### Description of work
- Wires up the `node--full` and `node--news--full` templates to implement the page title
- Wires related templates to provide the expected markup

### CODE REVIEW

File: templates/field/field--node--created.html.twig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{% include "@atoms/date-time/date-time.twig" with {
+  date_time__start: element['#object'].getCreatedTime|date('Y-m-d'),
```
Comment (nJim) [inline templates/field/field--node--created.html.twig:2]: I had to look this up as I didn't know what `element['#object']` would do. Glad this exists.

File: templates/field/field--node--uid.html.twig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{% for item in items %}
```
Comment (nJim) [inline templates/field/field--node--uid.html.twig:1]: I'm not sure what this template is for. There is a field_author that should have author names for the byline.

File: templates/user/username.html.twig [hunk 3]
```diff
@@ -0,0 +1 @@
+<span>{{ account.name.value }}</span>
```
Comment (nJim) [inline templates/user/username.html.twig:1]: I don't think we want to surface anything from the user object on this site. For news there is a field_author plain text field to be used on the byline.

File: templates/field/field--node--uid.html.twig (hunk 2 above)
Comment (ModulesUnraveled) [reply templates/field/field--node--uid.html.twig]: I didn't realize there was a field. I've switched everything over to use that.

File: templates/user/username.html.twig (hunk 3 above)
Comment (ModulesUnraveled) [reply templates/user/username.html.twig]: Deleted.

File: templates/node/node--news--full.html.twig [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{% include "@molecules/page-title/page-title.twig" with {
+  page_title__heading: label,
+  page_title__meta: 'By ' ~ content.field_author|render ~ date,
```
Comment (nJim) [inline templates/node/node--news--full.html.twig:3]: The author field is optional for news so we will want to make the 'By ' prefix conditional.

**Review by nJim — CHANGES_REQUESTED** [review]
Looks good. Only one small change on the new author stuff.

File: templates/node/node--news--full.html.twig (hunk 4 above)
Comment (ModulesUnraveled) [reply templates/node/node--news--full.html.twig]: Good catch. I pushed up a change that shows it if provided, and doesn't if it's not 👍

**Pull request merged**